install.sh

```diff
@@ -363,6 +363,66 @@ while true; do
     fi
 done
 
+validate_master_password() {
```

Comment:
Let us change the function name to `validate_opensearch_password`

Reply:
Can we try to reuse the existing function if possible?

Reply:
both have different conditions